diagnostic_updater/src/example.cpp

```diff
@@ -34,9 +34,10 @@
 
 #include <diagnostic_updater/diagnostic_updater.hpp>
 #include <diagnostic_updater/publisher.hpp>
-
 #include <std_msgs/msg/bool.hpp>
 
+#include "rclcpp/rclcpp.hpp"
+
```

Comment:
I see that it was probably wrong that this was missing before. But now it is technically a breaking change.

Reply:
Yes it is a breaking change. So this should not be merged into iron, but in rolling. I'll then get released for jazzy